examples/build.gradle

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2023, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.

```

Comment:
```suggestion
 * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.

```

Reply:
Fixed in [ad28983](https://github.com/ballerina-platform/module-ballerinax-ai.weaviate/pull/8/commits/ad2898359536f755c33720488b968161ab29f959)